config/locales/views/bgeigie_imports/awaiting_reponse/en-US.yml

```
@@ -0,0 +1,10 @@
+en-US:
+  bgeigie_imports:
+    awaiting_response:
+      index:
+        bgeigie_imports: bGeigie Imports
+        awaiting_response: Awaiting Reponse
```

"Response" is misspelled.
```suggestion
        awaiting_response: Awaiting Response
```